async-seq.js

```diff
@@ -0,0 +1,26 @@
+const fs    = require('fs');
+const path  = require('path');
+const async = require('async');
+const R = require('ramda');
```

Comment:
Let's not align the equals signs. Please match the style of the existing solutions as closely as possible, so we can focus on meaningful differences between the approaches without being distracted by stylistic differences.